cpp/ps0003/pooo3-soln.cpp

```diff
@@ -1,17 +1,23 @@
-# include <iostream>
+#include <iostream>
+
 using namespace std;
+
 int main(){
-int number;
-cout<<"Enter your number:";
-cin>> number;
-if (number%2==0)
-{
-cout<<"the number "<<number<<" is even";
+  // prompting user for number 2/2
+  int number;
+  cout << "Enter your number:";
+  cin >> number;
+  
+  // checking if the number is odd or even 3/3
+  if (number % 2 == 0)
+  {
+    cout << "the number " << number << " is even";
+  }
+  else 
+  {
+    cout << "the number " << number << " is odd";
+  }

```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console